tools/cldr-apps/src/test/java/org/unicode/cldr/unittest/web/TestUserRegistry.java

```diff
@@ -30,19 +30,19 @@ public void TestCanSetUserLevel() {
 
         User a = reg.new User(id++);
         a.userlevel = UserRegistry.ADMIN;
-        a.org = "SurveyTool";
+        a.org = Organization.surveytool.name();
 
         User b = reg.new User(id++);
         b.userlevel = UserRegistry.TC;
-        b.org = "Bangladesh";
+        b.org = Organization.bangladesh.name();
 
         User c = reg.new User(id++);
         c.userlevel = UserRegistry.MANAGER;
-        c.org = "Mozilla";
+        c.org = Organization.mozilla.name();
 
         User d = reg.new User(id++);
         d.userlevel = UserRegistry.VETTER;
-        d.org = "Bangladesh";
+        d.org = Organization.bangladesh.name();
```

Comment:
referring to Organization.bangladesh.name() instead of hard-coding "Bangladesh" ensures that the compiler will catch any invalid/outdated/misspelled/variant org names in our tests